packages/server/src/tests/banners/channelBannerTests.test.ts

```diff
@@ -20,18 +20,21 @@ describe('getDesignForVariant', () => {
         });
         const fooDesign = factories.bannerDesign.build({
             name: 'FOO',
-            image: { altText: 'Foo Alt' },
+            visual: { kind: 'Image', altText: 'Foo Alt' },
         });
         const barDesign = factories.bannerDesign.build({
             name: 'BAR',
-            image: { altText: 'Bar Alt' },
+            visual: { kind: 'Image', altText: 'Bar Alt' },
         });
         const designs: BannerDesignFromTool[] = [fooDesign, barDesign];
 
         const design = getDesignForVariant(variantWithDesign, designs);
```

Comment:
Am getting a permission denied when running these tests, presume its just my privileges

Reply:
that's strange, running `yarn test` works for me locally